db/migrate/20200311125719_create_sessions.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSessions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :sessions do |t|
+
+      t.timestamps
```

Comment:
Why this table has only a timestamp? What's the identifier? And also, shouldn't it have a foreign key to the `:user`?

Reply:
It looks as though this is a knock on from the alternate (and incorrect) method of creating models which I have been following up until this point. 

There is no foreign key to the user however as the create method validates the email of the user exists in the database, the password is valid and matches and the log_in method then sets a session id to match the user id. 

Maybe in another story we can revisit this functionality and improve it based on what I have learned since implementing it.